VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -114,6 +117,7 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
         );
         this.displayConfirmStartHearingPopup = false;
         this.hearingStartingAnnounced = true; // no need to play announcements for a judge
+        this.hearingVenueIsScottish$ = this.hearingVenueFlagsService.hearingVenueIsScottish$;
```

Comment:
Could we add this to the to the waiting room base component?

Reply:
@FarronF Sure, we can move it to the base class, though the class is already doing so much

Reply:
I do agree the waiting room base does too much but I think it's better to have this there rather than repeat ourselves over the child components.